### PR TITLE
DevKit updates for 3.x branch

### PR DESCRIPTION
**Pull request by SonataCI**

### CODE REVIEW

File: Tests/Route/CmsPageRouterTest.php [hunk 1]
```diff
@@ -163,6 +163,9 @@ public function testGenerateWithPageSlugValid()
         $this->assertEquals('//localhost/my/path?foo=bar', $url);
     }
 
+    /**
+     * @group legacy
+     */
```
Comment (greg0ire) [inline Tests/Route/CmsPageRouterTest.php]: @pix-art please add tests for the new way.

**Comment by core23** [comment]
This should fix travis: https://github.com/sonata-project/SonataDoctrineORMAdminBundle/pull/750

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -74,7 +74,7 @@ php-cs-fixer fix --verbose
 
 #### The documentation
 
-The documentation is mostly written with the `rst` format, and can be found in the `docs` directory.
+The documentation is mostly written with the `rst` format, and can be found in the `Resources/doc` directory.
```
Comment (core23) [inline CONTRIBUTING.md]: We should fix this on the dev-kit

**Comment by core23** [comment]
It looks like that the `weak_vendors` env entry has some problems when mocking `getMockForAbstractClass(Doctrine\ORM\QueryBuilder::class)`.

The class has has some `@deprecated` methods which causes the warnings. After removing the annotation, everything is fine.

cc @greg0ire

**Comment by greg0ire** [comment]
@core23 This might fix it: https://github.com/symfony/symfony/pull/24864

**Comment by core23** [comment]
So we must wait for phpunit bridge 3.4 to fix this issue?

**Comment by core23** [comment]
Maybe we should revert the `weak_vendors` mode.

**Comment by greg0ire** [comment]
Or should we mark the test with `@legacy`?

**Comment by core23** [comment]
This is not the only project that will fail. So we should revert his mode for now.

**Comment by greg0ire** [comment]
Yeah probably :(

**Pull request merged**